webapp/channels/src/components/channel_header_dropdown/mobile_channel_header_dropdown_animation.tsx

```diff
@@ -10,26 +10,25 @@ const ANIMATION_DURATION = 350;
 type Props = {
     children?: ReactNode;
     show: boolean;
-}
+};
 
-export default class MobileChannelHeaderDropdownAnimation extends React.PureComponent<Props> {
-    render() {
-        return (
-            <CSSTransition
-                in={this.props.show}
-                classNames='mobile-channel-header-dropdown'
-                enter={true}
-                exit={true}
-                mountOnEnter={true}
-                unmountOnExit={true}
-                timeout={{
-                    enter: ANIMATION_DURATION,
-                    exit: ANIMATION_DURATION,
-                }}
-            >
-                {this.props.children}
-            </CSSTransition>
-        );
-    }
-}
+const MobileChannelHeaderDropdownAnimation = ({show, children}: Props) => {
+    return (
+        <CSSTransition
+            in={show}
+            classNames='mobile-channel-header-dropdown'
+            enter={true}
+            exit={true}
+            mountOnEnter={true}
+            unmountOnExit={true}
+            timeout={{
+                enter: ANIMATION_DURATION,
+                exit: ANIMATION_DURATION,
+            }}
```

Comment:
Can we define this outside of the component? If not, on every render, we are passing a new value to `timeout`, potentially triggering a re-render underneath.